t/unit/backends/test_mongodb.py

```diff
@@ -1,9 +1,17 @@
 from __future__ import absolute_import, unicode_literals
 
 import datetime
+import sys
 from pickle import dumps, loads
 
 import pytest
+import pytz
+try:
+    import pymongo
+    import bson
+except ImportError:
+    pymongo = None
+    bson = None
```

Comment:
I'd probably use https://docs.pytest.org/en/latest/reference.html#pytest-importorskip-ref in the test that needs it:

```
     def fake_mongo_collection_patch(self, monkeypatch):
         bson = pytest.importorskip("bson")
```

Reply:
looks much better now, thanks.